pkg/exporter/parquet/parquet.go

```diff
@@ -1,50 +1,39 @@
 package parquet
 
 import (
-	"context"
 	"fmt"
 	"io"
 	"time"
 
 	"github.com/pkg/errors"
 	"github.com/thanos-community/obslytics/pkg/dataframe"
-	"github.com/thanos-community/obslytics/pkg/output"
+	"github.com/thanos-community/obslytics/pkg/exporter"
 	parquetwriter "github.com/xitongsys/parquet-go-source/writerfile"
 	"github.com/xitongsys/parquet-go/parquet"
 	"github.com/xitongsys/parquet-go/source"
 	"github.com/xitongsys/parquet-go/writer"
 )
 
-// Output implements output.Output interface.
-type Output struct{}
+var _ exporter.Encoder = &Encoder{}
```

Comment:
@bwplotka could you please explain the purpose of this line?

Reply:
Added comment (: